Popular Movies/Popular Movies/Controllers/RssFeedTableViewController.m

```diff
@@ -11,28 +11,32 @@
 #import "NewsTableViewCell.h"
 @interface RssFeedTableViewController ()
 {
-    NSMutableArray * lista_vijesti;
-    
+    NSMutableArray* newsArray;
+    NSXMLParser* newsParser;
+    News* singleArticle;
+    NSString* currentElement;
+    NSMutableString* newsTitle;
+    NSMutableString* newsText;
+    NSMutableString* newsSource;
+    IBOutlet UITableView *newsTableView;
 }
 @end
 
 @implementation RssFeedTableViewController
-NSString *const newsCellIdentifier = @"newsCell";
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    //registrating the custom table view
+    self.tableView.userInteractionEnabled = YES;
+    newsTableView.scrollEnabled = YES;
+    //Registrating the custom table view with reuse identifier
     [self.tableView  registerNib:[UINib nibWithNibName:NSStringFromClass([NewsTableViewCell class]) bundle:nil] forCellReuseIdentifier:NSStringFromClass([NewsTableViewCell class])];
-    //this part of the code needs to be replaced with populating the table view with API data
-    lista_vijesti = [[NSMutableArray alloc]init];
-    News * news1 = [[News alloc] init];
-    news1.title = @"Music For Self Improvment";
-    news1.source_link = @"www.source.com";
-    news1.text = @"When I was just starting 6th grade I got my first job. Paperboy! Boy, was I excited. At that time I had spent a lot of time actually playing the video game Paperboy, so I knew I had what it took to get the job done. But, its just not that easy.";
-    [lista_vijesti addObject:news1];
-    [lista_vijesti addObject:news1];
-    [lista_vijesti addObject:news1];
-
+    newsArray = [[NSMutableArray alloc]init];
```

Comment:
This shouldn't be part of controller. Create NewsServices class and put there methods that will fetch news, so you can load news in one line from viewDidLoad.